### ISSUE TITLE
The "id" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by LuckeeDev** [nxext/nx-extensions#241]
**Describe the bug**
I'm getting this error after creating a project with `@nxext/stencil` and setting up a Storybook instance for my library.
```
> nx run components:storybook 
The "id" argument must be of type string. Received undefined

———————————————————————————————————————————————

>  NX   ERROR  Running target "components:storybook" failed

  Failed tasks:

  - components:storybook
```

**To Reproduce**
- Bootstrap a new project with `yarn create nx-workspace`;
- Install `@nxext/stencil`;
- Create a new library and set up a Storybook configuration for this library;
- Try and launch the Storybook instance.

**Additional context**
NX Workspace is at version 12 and `@nxext/stencil` is at the latest version.

**Comment by DominikPieper** [comment]
Hey, thanks for the report. I'm aware of that problem and working on it already. The next release will contain a fix for that, hopefully.

**Comment by DominikPieper** [comment]
Probably I'll remove the feature temporarily. With storybook v6 the Nx executor takes a parameter for the uiFramework and right now there's just Angular and React as a working option. It has to be changed to support HTML as well, with v5 this wasn't a problem.

**Comment by JoMen6** [comment]
This could be fix it nrwl/nx#5610 :tada:

**Comment by DominikPieper** [comment]
Yes, it should be fixed with that PR :-)
